docker-compose-installed.yaml

```diff
@@ -65,7 +66,7 @@ services:
       - '28015:28015'
 
   validator:
```

Comment:
Likewise, `supply-validator`?

Reply:
This seems reasonable. I didn't change it because there will never be a `supply-validator`, but just for consistency, I could make that change.

Reply:
Perhaps we need to have a larger conversation about service/image/container naming conventions. The naming convention I was using was is `{library name}-{component name}`. In this, that would mean `sawtooth-validator`, though I skipped it because there will never be any other kind of validator, and so it felt like an extraneous change.

I don't believe we need to worry about external conflicts with service names, just within the compose file itself. However, I should probably modify the container names, as I neglected to change those.

Reply:
Yeah, I think I was equating service names and container names.  I think just ensuring that the container names won't conflict with the other sawtooth docker compose files would be a good idea.